src/legacy/server/saved_objects/service/lib/repository.js

```diff
@@ -663,6 +683,15 @@ export class SavedObjectsRepository {
     }
   }
 
+  _getIndexForType(type) {
+    return (
+      (this._schema.definition &&
+        this._schema.definition[type] &&
+        this._schema.definition[type].indexPattern) ||
```

Comment:
Although JS will let you get away with it, you're accessing private properties of a Typescript class